Documentation/Books/Manual/Architecture/SingleInstanceVsCluster.md

```diff
@@ -0,0 +1,157 @@
+Single Instance vs. Cluster
+===========================
+
+In general, a single server configuration and a cluster configuration
+of ArangoDB behave very similarly. However, there are difference due to
+the different nature of these setups. This can lead to different behavior
+between these two configurations. A summary of potential differences follows.
+
+See [Migrating from a Single Instance to a Cluster](../Deployment/MigratingSingleInstanceCluster.md)
+for practical information.
+
+Locking and dead-lock prevention
+--------------------------------
+
+In a single server configuration all data is local and dead-locks can
```

Comment:
this applies only to AQL queries

Reply:
What about API endpoints such as document? Or does everything use AQL internally?

Reply:
@graetzer Can you edit the sentence please, as I am not sure what to do exactly.